tests/TestCase/ORM/Behavior/TranslateBehaviorTest.php

```diff
@@ -92,8 +92,8 @@ public function testCustomTranslationTable()
         $i18n = $items->getByProperty('_i18n');
 
         $this->assertEquals('\TestApp\Model\Table\I18nTable', $i18n->name());
+        $this->assertInstanceOf('TestApp\Model\Table\I18nTable', $i18n->target());
         $this->assertEquals('custom_i18n_table', $i18n->target()->table());
-        $this->assertEquals('test_custom_i18n_datasource', $i18n->target()->connection()->configName());
```

Comment:
While checking connection name might not be necessary for this particular test, [test_custom_i18n_datasource](https://github.com/cakephp/cakephp/blob/master/tests/bootstrap.php#L100) is defined in tests/bootstrap.php